tests/test_freshness_date_parser.py

```diff
@@ -1066,6 +1097,14 @@ def test_normalized_relative_dates(self, date_string, ago, period):
 
     @parameterized.expand([
         # English dates
+        param('in a fortnight', in_future={'days': 14}, period='day'),
+        param('next fortnight', in_future={'days': 14}, period='day'),
```

Comment:
I am not sure, but do we support `coming fortnight`?

Reply:
@gutsytechster Done this change, Please Look into it 😊.